.gitignore

```diff
@@ -1,5 +1,11 @@
 *.class
 *.log
+.idea
+.idea/*
+*.iml
+.#*
+Makefile
+.DS_Store
```

Comment:
These mac and editor-specific entries belong in your `~/.global_gitignore` (https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

Reply:
This file is a **project filter**: these are files the project doesn't want included. No matter what machine they occur on, they shouldn't be included in the project.
Incidentally, I copied these entries from the Processors .gitignore.

Reply:
I would also prefer to remove this. Other than that LGTM

Reply:
If that's what you guys want, I'll do it but I don't understand your thinking. One purpose of a project .gitignore file is to prevent accidental checkins of spurious files. Since many developers (including us) work on Macs, it doesn't seem unreasonable to prevent these types of unwanted files from accidentally being committed. Note that we already screen these types files from the Processors and Reach projects.